crates/rpc/src/tests.rs

```diff
@@ -1,17 +1,29 @@
 use std::{net::SocketAddr, time::Duration};
 
+use miden_air::{ExecutionProof, Felt, HashFunction};
```

Comment:
`Felt` could come from `miden_objects` and `ExecutionProof` could come from `miden_tx`.